tests/unit/array/sort-test.js

```diff
@@ -0,0 +1,83 @@
+import { sort } from 'ember-awesome-macros/array';
```

Comment:
Did you see the existing tests at https://github.com/kellyselden/ember-awesome-macros/blob/master/tests/integration/array/sort-test.js?